src/data/set/basic.lean

```diff
@@ -2011,6 +2017,13 @@ lemma surjective.nonempty_preimage (hf : surjective f) {s : set β} :
   (f ⁻¹' s).nonempty ↔ s.nonempty :=
 by rw [← nonempty_image_iff, hf.image_preimage]
 
+lemma surjective.nonempty [nonempty β] (hf : surjective f) : nonempty α :=
```

Comment:
This should live in `logic.nonempty`. And I think that it is sufficiently obvious to prove it in 1line.
```lean
let ⟨y⟩ := _i, ⟨x, hx⟩ := hf y in ⟨x⟩
```